### ISSUE TITLE
stopped working completely, just me?

### ISSUE THREAD

**Issue by mani2200** [devhotteok/TwitchLink#89]
it gives me an unknown error then when i click it, it says channel deleted or temporary error.

**Comment by mani2200** [comment]
i am talking about the scheduled downloads

**Comment by slamdunk95** [comment]
I'm getting the error also, but it's only affecting new downloads. I have a stream that was already downloading and it's still working, but any new scheduled download or download initiated from the search get the error.

**Comment by winterlady26** [comment]
I've been experiencing the same phenomenon since about 3hours ago. Changing the settings and rebooting the computer won't solve it.

**Comment by saleesep** [comment]
It's a change on Twitch's end that broke everything

**Comment by Fijxu** [comment]
This issue is highly related to this other issue
TL;DR: Twitch changed the way in which the playlist of the streams are obtained, making it more difficult to bypass ads and breaking every third party tool

https://github.com/streamlink/streamlink/issues/5370

**Comment by devhotteok** [comment]
Duplicate of #83
